x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapper.java

```diff
@@ -151,7 +152,8 @@ public class SemanticTextFieldMapper extends FieldMapper implements InferenceFie
     );
 
     public static final String CONTENT_TYPE = "semantic_text";
-    public static final String DEFAULT_ELSER_2_INFERENCE_ID = DEFAULT_ELSER_ID;
+    public static final String DEFAULT_FALLBACK_ELSER_INFERENCE_ID = DEFAULT_ELSER_ID;
```

Comment:
You have good comments about which ones you're falling back to, you could probably just use the static imports without re-assigning them